code/model/model_flags.h

```diff
@@ -47,7 +47,9 @@ namespace Model {
         Autorepair_if_disabled,		// Allows the subsystem to repair itself even if disabled (MageKing17)
         No_autorepair_if_disabled,	// Inversion of the previous; disallows this particular subsystem if the ship-wide flag is set (MageKing17)
         Share_fire_direction,		// (DahBlount) Whenever the turret fires, make all firing points fire in the same direction.
-		
+        No_sparks,          // Subsystem does not generate sparks if hit - m!m
+        No_debris_spawn,    // Don't spawn the small debris on impact - m!m
```

Comment:
I suggest renaming the second flag to `No_impact_debris`, both to match the text and also to be a little clearer.  (I initially thought this had something to do with live debris.)

Reply:
Thanks, that is a much better name for that.